docs/content/docs/connectors/datastream/kinesis.md

```diff
@@ -27,644 +27,348 @@ under the License.
-
-#### Enhanced Fan-Out Record Publisher
-
-- *[SubscribeToShard](https://docs.aws.amazon.com/kinesis/latest/APIReference/API_SubscribeToShard.html)*: this is called
-by per shard consuming threads to obtain shard subscriptions. A shard subscription is typically active for 5 minutes, 
-but subscriptions will be reaquired if any recoverable errors are thrown. Once a subscription is acquired, the consumer
-will receive a stream of [SubscribeToShardEvents](https://docs.aws.amazon.com/kinesis/latest/APIReference/API_SubscribeToShardEvent.html)s.
-Retry and backoff parameters can be configured using the `ConsumerConfigConstants.SUBSCRIBE_TO_SHARD_*` keys.
-
-- *[DescribeStreamSummary](https://docs.aws.amazon.com/kinesis/latest/APIReference/API_DescribeStreamSummary.html)*: this is called 
-once per stream, during stream consumer registration. By default, the `LAZY` registration strategy will scale the
-number of calls by the job parallelism. `EAGER` will invoke this once per stream and `NONE` will not invoke this API. 
-Retry and backoff parameters can be configured using the 
-`ConsumerConfigConstants.STREAM_DESCRIBE_*` keys.
+The `FlinkKinesisConsumer` is the deprecated connector based off the old source interface. We do not recommend users
+to use this.
 
-- *[DescribeStreamConsumer](https://docs.aws.amazon.com/kinesis/latest/APIReference/API_DescribeStreamConsumer.html)*:
-this is called during stream consumer registration and deregistration. For each stream this service will be invoked 
-periodically until the stream consumer is reported `ACTIVE`/`not found` for registration/deregistration. By default,
-the `LAZY` registration strategy will scale the number of calls by the job parallelism. `EAGER` will call the service 
-once per stream for registration only. `NONE` will not invoke this service. Retry and backoff parameters can be configured using the 
-`ConsumerConfigConstants.DESCRIBE_STREAM_CONSUMER_*` keys.  
-
-- *[RegisterStreamConsumer](https://docs.aws.amazon.com/kinesis/latest/APIReference/API_RegisterStreamConsumer.html)*: 
-this is called once per stream during stream consumer registration, unless the `NONE` registration strategy is configured.
-Retry and backoff parameters can be configured using the `ConsumerConfigConstants.REGISTER_STREAM_*` keys.
-
-- *[DeregisterStreamConsumer](https://docs.aws.amazon.com/kinesis/latest/APIReference/API_DeregisterStreamConsumer.html)*: 
-this is called once per stream during stream consumer deregistration, unless the `NONE` or `EAGER` registration strategy is configured.
-Retry and backoff parameters can be configured using the `ConsumerConfigConstants.DEREGISTER_STREAM_*` keys.  
+Note that there is no state compatibility between the `FlinkKinesisConsumer` and `KinesisStreamsSource`. 
+To migrate, consider starting the `KinesisStreamsSource` from `AT_TIMESTAMP` slightly before the time when the `FlinkKinesisConsumer` was stopped. 
+Note that this will result in some re-processed some records.
```

Comment:
nit: Note, this may result in some re-processed records.

Reply:
done